tests/Agent/UnitTests/Core.UnitTest/Utilities/RuntimeEnvironmentInfoTests.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using NUnit.Framework;
+
+namespace NewRelic.Agent.Core.Utils
+{
+    [TestFixture]
+    public class RuntimeEnvironmentInfoTests
+    {
+        [Test]
+        public void RuntimeEnvironmentInfo_OperatingSystem_ReportsWindows()
+        {
+            Assert.AreEqual("Windows", RuntimeEnvironmentInfo.OperatingSystem);
```

Comment:
Hmm... This will fail on Linux for anyone that runs the tests on Linux. Perhaps this should just be a not null check. Alternatively it looks like you can filter the testcase with NUnit:
https://docs.nunit.org/articles/nunit/writing-tests/attributes/platform.html

Reply:
Does anyone run our unit tests on Linux? I'll add a `[Platform("Win")]` attribute just in case. 

Really want to see about running our unit tests on Linux as well as Windows in our CI workflow - maybe I'll look into that.

Reply:
Nice fix. Running on linux is not part of our common workflow, but if/when it does become part of that workflow, it will be good to not instantly have failing tests. This is almost like an integration test as it depends on the OS which is generally abstracted by .NET (core atleast).